webrtc/RTCPeerConnection-set-description.html

```diff
@@ -0,0 +1,395 @@
+<!doctype html>
+<title>RTCPeerConnection Set Description tests</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script type="text/javascript">
+  'use strict';
+
+  const audioLineRegex = /\r\nm=audio.+\r\n/g
+  const videoLineRegex = /\r\nm=video.+\r\n/g
+
+  function countAudioLine(sdp) {
+    const matches = sdp.match(audioLineRegex);
+    if(matches === null) {
+      return 0;
+    } else {
+      return matches.length;
+    }
+  }
+
+  function countVideoLine(sdp) {
+    const matches = sdp.match(videoLineRegex);
+    if(matches === null) {
+      return 0;
+    } else {
+      return matches.length;
+    }
+  }
+
+  function similarMediaDescriptions(sdp1, sdp2) {
+    if(sdp1 === sdp2) {
+      return true;
+    } else if(countAudioLine(sdp1) !== countAudioLine(sdp2)) {
+      return false;
+    } else if(countVideoLine(sdp1) !== countVideoLine(sdp2)) {
+      return false;
+    } else {
+      return true;
+    }
+  }
+
+  function assert_defined(object) {
+    if(object === undefined || object === null)
+      assert_unreached('object must not be undefined or null');
+  }
+
+  function isSimilarSdp(sdp1, sdp2) {
+    assert_defined(sdp1);
```

Comment:
I think it is worth testing that.
Ideally, this should be isolated in its own test so that Chromium would only fail that test and would not fail all tests because of that.